packages/atlas-experiment-table/src/search.js

```diff
@@ -0,0 +1,52 @@
+import _ from 'lodash'
+
+// Don’t search for queries that are shorter than this
+const TOKEN_MIN_LENGTH = 3
```

Comment:
Any reason why we have specifically chosen 3 as a number for search length? I personally prefer the way the functionality is now at the moment that as soon as I type a single character is searches for that.

Reply:
Usually one or two letter search queries mean very little. One letter is definitely meaningless, since we’re matching any word that will contain that letter. Two-letter words are arguably similar. But three letters is a lot more discriminatory. For example Solr does this by default, and I think it’s in general a good thing. We can change it if users become confused.

Reply:
Okay. This makes sense.